cpu/stm32/include/periph/wl/Kconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2021 Freie Universitaet Berlin
```

Comment:
This would fit better either on in the `kconfigs` directory or as part of an existing Kconfig file there.

Reply:
Okay